engine/common/follower/cache/cache.go

```diff
@@ -305,6 +320,7 @@ func (c *Cache) cache(blockID flow.Identifier, block *flow.Block) (equivocation
 	if !added { // future proofing code: we allow an overflowing HeroCache to potentially eject the newly added element.
 		return
 	}
+	stored = true
```

Comment:
would suggest to assign the return value of https://github.com/onflow/flow-go/blob/aa4d24b0b9d098219b67211b345795f43bb71b87/engine/common/follower/cache/cache.go#L319 directly to `stored`

[Lines 319-L323](https://github.com/onflow/flow-go/blob/aa4d24b0b9d098219b67211b345795f43bb71b87/engine/common/follower/cache/cache.go#L319-L323) would then be
```golang
	stored = c.backend.Add(blockID, block)
	if !stored { // future proofing code: we allow an overflowing HeroCache to potentially eject the newly added element.
		return
	}
```